static/css/training.css

```diff
@@ -139,20 +81,77 @@ section.call-to-action .main-section .cta-image > img {
   text-decoration: none;
 }
 
+.mainsection {
+  background-repeat: no-repeat;
+  background-size: 100%;
+}
+
+.second {
+  padding-top:4%;
+}
+
 h5 {
   font-size: 16px;
   font-weight: 500;
   line-height: 1.5em;
   margin-bottom: 2em;
 }
 
+.logo-certification .cta-image .cta-image-after img {
+  width:300px !important;
+}
+
 .white-text {
   color: #fff;
 }
 
 .padded {
-  padding-top: 100px;
-  padding-bottom: 100px;
+  padding-top: 3%;
+  padding-bottom: 4%;
+}
+
+.padded-bottom-banner {
+  padding-top:0%;
+  padding-bottom: 2%;
+}
+
+.half {
+  width:50%;
+  float:left;
+  padding-left:10%;
+  padding-right:10%;
+  padding-top:4%;
+  padding-bottom:5%;
+}
+
+.half-image{
```

Comment:
**Part 3:** Let's keep this class as minimal as possible, to avoid weird conflicts.

```
.half-image {
  width:50%;
  float:left;
}
```

Note for learning purposes: you could also use `float: right` here and it would do basically the same thing.